src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -191,9 +196,15 @@ def create_standardised_results(
     default="descending",
     show_default=True,
 )
+@click.option(
+    "--phenotype-only",
+    type=bool,
+    default=False,
+    help="Specify if Exomiser was run with phenotype-only analysis.",
+)
 def post_process_exomiser_results(
```

Comment:
What was the motivation to have a CLI for this? Shouldnt postprocessing be dealt with by pheval run? Or is this something internal for the exomiser team that is not relevant for PhEval?

Reply:
It's something internal for the Exomiser team that's not relevant for PhEval. When I do the benchmarking for Exomiser I use the HPC and as I haven't implemented an option to pheval run on the environment yet it's easier to use the separated commands for now

Reply:
Cool thank you!